tests/integration/test_action.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Integration test for actions."""
+
+import pytest
+from juju.application import Application
+
+from .conftest import TEST_EXTERNAL_HOSTNAME_CONFIG
+
+
+@pytest.mark.abort_on_fail
+async def test_get_certificate_action(
+    configured_application_with_tls: Application,
+):
+    """Deploy the charm with valid config and tls integration.
+
+    Assert on valid output of get-certificate.
```

Comment:
Perhaps one of our linters is missing? These tests should be described in the arrange/act/assert format which is checked for using https://github.com/jdkandersson/flake8-test-docs

Reply:
I just double-checked and we do have `flake8-test-docs` installed in the `lint` step : https://github.com/canonical/haproxy-operator/blob/6445921388de49dee8fc1a29fda0d4e9dc0b7672/tox.ini#L44

Reply:
I updated the docstring of integration tests to follow the format. I haven't looked into why the plugins is not working properly yet, but I'll find the time to look into it.